fastestimator/backend/_binary_crossentropy.py

```diff
@@ -14,13 +14,12 @@
 # ==============================================================================
 from typing import Dict, Optional, TypeVar
 
-import tensorflow as tf
 import torch
 
 from fastestimator.backend._reduce_mean import reduce_mean
 
-Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
```

Comment:
We could remove the `TypeVar` tensor here and just use `torch.Tensor` for typing